From bef5599b2f7bc2154e9cb2fa09c17d28ed3cf642 Mon Sep 17 00:00:00 2001
From: Steve deRosier <steve.derosier@lairdtech.com>
Date: Sat, 9 Apr 2016 11:38:08 -0700
Subject: [PATCH] laird: wbXXn: add Micrel phy chips to the wb defconfigs

We want to support Micrel phy chips. In particular, we want to support
the KSZ8081RNACA phy as well as the KSZ8895 switch chip.

Bug: 9292
---
 arch/arm/configs/wb45n_defconfig | 2 ++
 arch/arm/configs/wb50n_defconfig | 1 +
 2 files changed, 3 insertions(+)

diff --git a/arch/arm/configs/wb45n_defconfig b/arch/arm/configs/wb45n_defconfig
index 28a94d3..dd4a163 100644
--- a/arch/arm/configs/wb45n_defconfig
+++ b/arch/arm/configs/wb45n_defconfig
@@ -159,6 +159,8 @@ CONFIG_MACB=y
 # CONFIG_NET_VENDOR_STMICRO is not set
 # CONFIG_NET_VENDOR_WIZNET is not set
 CONFIG_ICPLUS_PHY=y
+CONFIG_MICREL_PHY=y
+CONFIG_MICREL_KS8995MA=m
 CONFIG_PPP=m
 CONFIG_PPP_BSDCOMP=m
 CONFIG_PPP_DEFLATE=m
diff --git a/arch/arm/configs/wb50n_defconfig b/arch/arm/configs/wb50n_defconfig
index f831f55..112c297 100644
--- a/arch/arm/configs/wb50n_defconfig
+++ b/arch/arm/configs/wb50n_defconfig
@@ -159,6 +159,7 @@ CONFIG_MACB=y
 # CONFIG_NET_VENDOR_WIZNET is not set
 CONFIG_ICPLUS_PHY=y
 CONFIG_MICREL_PHY=y
+CONFIG_MICREL_KS8995MA=m
 CONFIG_PPP=m
 CONFIG_PPP_BSDCOMP=m
 CONFIG_PPP_DEFLATE=m
-- 
1.9.1

